examples/tv-casting-app/darwin/TvCasting/TvCasting/CertTestViewModel.swift

```diff
@@ -0,0 +1,383 @@
+/**
+ *
+ *    Copyright (c) 2020-2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+//
+//  CertTestViewModel.swift
+//  TvCasting
+//
+//  Created by Jain, Amit on 1/10/23.
+//
```

Comment:
Can you delete this from here and any other file it was added to?

Reply:
Done.